### PR TITLE
added scrollview to consent components to improve UX on zoomed screens

### PR DESCRIPTION
**Pull request by mgawron-fp**

### CODE REVIEW

File: Xamarin.UserConsent/UI/ConsentFormPage.xaml [hunk 1]
```diff
@@ -19,34 +19,36 @@
     <Frame StyleClass="consentPopup"
            VerticalOptions="Center"
            HorizontalOptions="Center">
-        <StackLayout VerticalOptions="Center" 
+        <ScrollView>
```
Comment (mgawron-fp) [inline Xamarin.UserConsent/UI/ConsentFormPage.xaml:22]: FYI - this is the important addition

File: Xamarin.UserConsent/UI/RecordingConsentView.xaml [hunk 2]
```diff
@@ -1,76 +1,79 @@
...
     <ContentView.Content>
-        <StackLayout
-          BindingContext="{x:Reference RecordingConsent}"
-          StyleClass="mainLayout">
-            <Label
-                Text="{Binding Header}"
-                StyleClass="headerLabel"/>
-            <BoxView
-                StyleClass="boxView"/>
+        <ScrollView>
```
Comment (mgawron-fp) [inline Xamarin.UserConsent/UI/RecordingConsentView.xaml:11]: FYI - this is the important addition

File: Xamarin.UserConsent/UI/ConsentFormPage.xaml (hunk 1 above)
Comment (rnowosielski) [reply Xamarin.UserConsent/UI/ConsentFormPage.xaml:22]: @mgawron-fp you need to bump the version so that it gets published.

Comment (mgawron-fp) [reply Xamarin.UserConsent/UI/ConsentFormPage.xaml:22]: done

**Review by DorotaMacura — APPROVED** [review]

**Pull request merged**